File: financial/TimorINSS-MInterno/src/app/models/destinatario.ts
import { PagamentoExecutado } from "./pagamentos_executados";

export interface Destinatario
{
  id: number;
  entidadeFk?: number;
  trabalhadorFK?: number;
  niss: string;
  tin: string;
  nome: string;
  morada: string;
}

File: backend/TimorINSS-BackEnd-main/TimorINSS-BackEnd/ExcelReaderService/Models/Destinatario.cs
using CsvHelper.Configuration.Attributes;

namespace TimorINSSBackEnd.ExcelReaderService.Models
{
    public class Destinatario: Pagamento
    {
        [Name("Niss")]
        public string Niss { get; set; }

        [Name("Tin")]
        public string Tin { get; set; }

        [Name("Name")]
        public string Name { get; set; }

        [Name("Address")]
        public string Address { get; set; }
    }

    public class Pagamento
    {

        [Name("IBAN")]
        public string IBAN { get; set; }

        [Name("Swift")]
        public string Swift { get; set; }

        [Name("Account Number")]
        public string AccountNumber { get; set; }

        [Name("Amount")]
        public decimal? Amount { get; set; }

    }
}
